libraries/AP_OpticalFlow/OpticalFlow.h

```diff
@@ -80,6 +80,7 @@ class OpticalFlow
 
 private:
     OpticalFlow_backend *backend;
+    AP_AHRS_NavEKF &_ahrs;
```

Comment:
for future... we may want to add a method to AP_AHRS_NavEKF to get a reference, so we don't need to keep passing it in constructors.